.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file shouldn't be in here
it is best to add `.idea` to your global htaccess